### DISCUSSION TITLE
Does CEFSharp support Nuance Power Mic Web Extension

### DISCUSSION THREAD

**Discussion by email2mohanraj** [cefsharp/CefSharp#4755] [General]
Hi,

I am trying to access this extension using CEFSharp in WPF application and it was not supported, let me know how to make this extension to work

https://chromewebstore.google.com/detail/nuance-powermic-web-exten/fmiojochalhealflohaicjncoofdjjfb?pli=1

**Comment by amaitland** [comment]
> I am trying to access this extension using CEFSharp in WPF application and it was not supported, let me know how to make this extension to work

Not supported.

You can only use extensions via the `Chrome Runtime`. See https://github.com/cefsharp/CefSharp/discussions/4123

(No support for loading extensions programmatically currently.)
